.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request:
+    types: [ opened, synchronize ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  lint:
+    name: Lint
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.15.0'
+
+      # Runs a single command using the runners shell
+      - name: Node version
+        run: node -v
+      - name: Linter
+        run: yarn lint
```

Comment:
# Runs a single command using the runners shell
      - name: Node version
        run: node -v
      - name: Install dependencies
        run: yarn install
      - name: Linter
        run: yarn lint